Applied-Research-Winter-2022/Pozter
# Poz 81 upload user content to cloudinary

## Link Jira ticket:
https://jess-nguy.atlassian.net/browse/POZ-81

## Author Notes:
user contents can now be uploaded to Cloudinary

## Preview:
![88e085e4-d701-4eac-a6e7-cf3ed3fa80c5](https://user-images.githubusercontent.com/62397382/160360760-f7eea8ed-94c8-4fb6-aac1-2e5d6d9ac464.jpg)

## Impacts and Notes:
the image URL is not saved in the config object of the user_content record yet

## Testing Guidance:
- run frontend and server
- go to [UserContent](http://localhost:8080/create/3) and upload an image (click the link to see it)
- log in to Cloudinary to see the [media library](https://cloudinary.com/console/c-bed08198b4d3d2c448af6068756a72/media_library/folders/bfe7a06db3896af3fc8fce090f862400e9)

## Author Checklist:

- [x] Change Jira ticket status to into Review.
- [n/a] Updated the documentation.
